addons/l10n_de_skr03/migrations/3.1/post-migrate_update_taxes.py

```diff
@@ -1,6 +1,8 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
-from odoo.addons.account.models.chart_template import update_taxes_from_templates
+from odoo import api, SUPERUSER_ID
 
 
 def migrate(cr, version):
-    update_taxes_from_templates(cr, 'l10n_de_skr03.l10n_de_chart_template')
+    env = api.Environment(cr, SUPERUSER_ID, {})
+    for company in env['res.company'].search([('chart_template', '=', 'de')]):
```

Comment:
shouldn't it be like this? same for the other one
```suggestion
    for company in env['res.company'].search([('chart_template', '=', 'de_skr03')]):
```

Reply:
> shouldn't it be like this? same for the other one

Yes. My bad. Thanks for the review. Changes has been done.